src/Kafka/Avro/SchemaRegistry.hs

```diff
@@ -223,10 +235,22 @@ wrapError someErr = case fromException someErr of
   Just httpErr -> fromHttpError httpErr (\_ -> SchemaRegistrySendError (displayException someErr))
```

Comment:
Not directly related to this PR, but catching and wrapping every possible `Exception` could be dangerous.

Reply:
Yeah, we should only be catching Http/Net exceptions and not the others...